R/standardize_date.R

```diff
@@ -106,11 +106,11 @@
 #'   )
 #' )
 standardize_dates <- function(data,
-                              target_columns  = NULL,
-                              format          = NULL,
-                              timeframe       = NULL,
-                              error_tolerance = 0.5,
-                              orders          = NULL) {
+                              target_columns = NULL,
+                              format = NULL,
+                              timeframe = NULL,
+                              error_tolerance = 0.4,
```

Comment:
Is this value changed on purpose?

Reply:
Yes - `error_tolerance` had been annouced to be set at `0.4` by default. But the initial threshold of `0.5` was kept. 
I updated it when I was going through function documentation.